pkg/server/plugin/nodeattestor/repository.go

```diff
@@ -0,0 +1,21 @@
+package nodeattestor
+
+type Repository struct {
+	NodeAttestors map[string]NodeAttestor
+}
+
+func (repo *Repository) GetNodeAttestorNamed(name string) (NodeAttestor, bool) {
```

Comment:
Will `GetNodeAttestor` suffice? There is only one key for the map at present, and it will likely remain the primary key?

Reply:
I'd be fine with that name change, but tried not to modify the (agent|server).Catalog interfaces as part of this change.

Reply:
If we're going to change the interfaces (even superficially), I'd rather it happen on another PR. It don't want to add call-site updates to this already monstrous changeset :)